sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/DotCoverReportParserTest.java

```diff
@@ -101,6 +147,18 @@ public void valid() throws Exception {
       .startsWith("dotCover parser: found coverage for line '12', hits '0' when analyzing the path '");
   }
 
+  @Test
+  public void valid_big() throws Exception {
+    Coverage coverage = new Coverage();
+    new DotCoverReportParser(alwaysTrue).accept(new File("src/test/resources/dotcover/valid_big.html"), coverage);
+
+    String filePath = new File("mylibrary\\calc.cs").getCanonicalPath();
+    assertThat(coverage.files()).containsOnly(filePath);
+    assertThat(coverage.hits(filePath)).hasSize(10000);
+    assertThat(logTester.logs(LoggerLevel.INFO).get(0)).startsWith("Parsing the dotCover report ");
```

Comment:
could you check the number of `TRACE` statements? and also the number of `INFO` statements (to be clear if there's only one or more TRACE/INFO log lines)